[2025-04-01T17:31:35.482+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-01T17:31:35.509+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_duckdb.query_duckdb manual__2025-04-01T17:31:27.854387+00:00 [queued]>
[2025-04-01T17:31:35.533+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_duckdb.query_duckdb manual__2025-04-01T17:31:27.854387+00:00 [queued]>
[2025-04-01T17:31:35.540+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-01T17:31:35.563+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): query_duckdb> on 2025-04-01 17:31:27.854387+00:00
[2025-04-01T17:31:35.578+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2480) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-01T17:31:35.580+0000] {standard_task_runner.py:72} INFO - Started process 2489 to run task
[2025-04-01T17:31:35.581+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_duckdb', 'query_duckdb', 'manual__2025-04-01T17:31:27.854387+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/etl_duckdb.py', '--cfg-path', '/tmp/tmp87z61wxq']
[2025-04-01T17:31:35.582+0000] {standard_task_runner.py:105} INFO - Job 13: Subtask query_duckdb
[2025-04-01T17:31:35.643+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_duckdb.query_duckdb manual__2025-04-01T17:31:27.854387+00:00 [running]> on host 1df6dff01eb9
[2025-04-01T17:31:35.756+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_duckdb' AIRFLOW_CTX_TASK_ID='query_duckdb' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T17:31:27.854387+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-01T17:31:27.854387+00:00'
[2025-04-01T17:31:35.757+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-01T17:31:35.758+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-01T17:31:35.759+0000] {logging_mixin.py:190} INFO - Current task name:query_duckdb state:running start_date:2025-04-01 17:31:35.510369+00:00
[2025-04-01T17:31:35.759+0000] {logging_mixin.py:190} INFO - Dag name:etl_duckdb and current dag run status:running
[2025-04-01T17:31:35.760+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-01T17:31:35.838+0000] {logging_mixin.py:190} INFO - Gagal menjalankan query ke DuckDB: IO Error: Could not set lock on file "/opt/***/data/database.duckdb": Conflicting lock is held in PID 0. See also https://duckdb.org/docs/connect/concurrency
[2025-04-01T17:31:35.839+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-01T17:31:35.852+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-01T17:31:35.853+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_duckdb, task_id=query_duckdb, run_id=manual__2025-04-01T17:31:27.854387+00:00, execution_date=20250401T173127, start_date=20250401T173135, end_date=20250401T173135
[2025-04-01T17:31:35.883+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-01T17:31:35.883+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-01T17:31:35.884+0000] {logging_mixin.py:190} INFO - Dag name:etl_duckdb queued_at:2025-04-01 17:31:27.988654+00:00
[2025-04-01T17:31:35.884+0000] {logging_mixin.py:190} INFO - Task hostname:1df6dff01eb9 operator:PythonOperator
[2025-04-01T17:31:35.917+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-01T17:31:35.940+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-01T17:31:35.944+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
